README.md

```diff
@@ -78,15 +78,20 @@ CriticalMAAS program.
 
 ### Geologic metadata curation
 
-- [`UW-Macrostrat/macrostrat-xdd`](https://github.com/UW-Macrostrat/macrostrat-xdd):
-  System to synthesize xDD-driven geologic unit descriptions from the scientific
-  literature (_placeholder for future development_)
+We hope to build better rock-record descriptions from the geological literature by:
+- Discovering concepts linked to known geological units
+- Finding new units based on proximity to known entities
+
+The starting point for this is two exploratory projects by UW-Madison computer science students, who are both taking different approaches to the problem:
 - [`UW-Macrostrat/factsheet-generator`](https://github.com/UW-Macrostrat/factsheet-generator):
   An LLM-assisted generator for geological "fact sheets" that operates over the
   scientific literature (_Bill Xia_; Computer Science)
 - [`UW-Macrostrat/unsupervised-kg`](https://github.com/UW-Macrostrat/unsupervised-kg):
-  Knowledge graph construction to discover new geologic entities in the
-  scientific literature (_Devesh Sarda_; Computer Science)
+  Unsupervised knowledge graph construction to discover new entitities from gelogical literature (_Devesh Sarda_; Computer Science)
+
+Eventually, both of these projects will be combined into a final system at [`UW-Macrostrat/macrostrat-xdd`](https://github.com/UW-Macrostrat/macrostrat-xdd). 
+
+We are also working on extract metadata from maps:
```

Comment:
What will we do with these links when we have them? Spell it out. they will be new filterable / searchable entities describing rocks, including their mineral contents, etc. Getting the full life cycle from What we're doing -> How -> **Why** is important to demonstrate value to the program and as a general practice.

Reply:
+1

Reply:
Pretty good — I would tie it back to CriticalMAAS as well by specifically stating that this will allow TA3 performers to filter b specific criteria extracted from the geologic literature.